vscode-extension/src/constants.ts

```diff
@@ -5,4 +5,4 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-export const SEMVER_RANGE = '>=13.3.x <14';
+export const SEMVER_RANGE = '>=13.3.x';
```

Comment:
There are no `13.3.x` releases, the last release in v13 was `13.2.0`. Also the README says the VSCode extension should work with relay-compiler v13 and above.

```suggestion
export const SEMVER_RANGE = '>=13.x';
```